apps/analysis_framework/migrations/0041_widget_mapping.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.2.17 on 2024-02-22 05:02
+from django.db import migrations
+import copy
+
+
+def analysis_framework_widgets_mapping(apps, schema_editor):
+    AnalysisFramework = apps.get_model('analysis_framework', 'AnalysisFramework')
+    af_qs = AnalysisFramework.objects.filter(properties__isnull=False)
+    for af in af_qs:
+        if af.properties == {}:
+            continue
+        af_prop = af.properties
+        # Remove super legacy config
+        af_prop.pop('old_stats_config', None)
+
+        # For reassurance
+        old_stats_config = copy.deepcopy(af_prop)
+
+        # Migrate legacy config to latest
+        # -- Widget1D
+        af_prop['stats_config']['widget_1d'] = (
+            af_prop['stats_config'].get('widget_1d') or
+            af_prop['stats_config'].get('widget1d') or
+            []
+        )
+
+        # -- Widget2D
+        af_prop['stats_config']['widget_2d'] = (
+            af_prop['stats_config'].get('widget_2d') or
+            af_prop['stats_config'].get('widget2d') or
+            []
+        )
```

Comment:
We need to use the old one as fallback
```suggestion
        af_prop['stats_config']['widget_2d'] = (
            af_prop['stats_config'].get('widget2d') or
            af_prop['stats_config'].get('widget_2d') or
            []
        )
```